_userdocs/Why-Choose-OpenStack.md

```diff
@@ -130,7 +130,7 @@ You might ask: Now that I know all of this, how do I actually *get* an OpenStack
 
 1. **As a collection of packages (called a ‘distribution’ or ‘distro’)**: The OpenStack website offers a standard, downloadable distribution of OpenStack. Apart from the default OpenStack distribution, many vendors offer proprietary distributions that include some value-added services. Distributions help with the installation and upgrade process of using OpenStack, which can be laborious. However, distributions are not particularly helpful with simplifying the operation of an OpenStack cloud. Customers who successfully use distributions usually have significant experience with OpenStack. Distributions are created by many vendors, as many as 25, including Mirantis, Red Hat, HP, Platform 9, and others. A list of distribution vendors is available in the OpenStack Marketplace, as is a list of consultants who might help you with an OpenStack deployment.
 
-2. **As a service**: OpenStack as a service means that you purchase your (private) cloud services on contract from experienced cloud operators, such as Blue Box or Rackspace. Cloud operators provide support to their clients, so that you are free to work “with” OpenStack, not “on” OpenStack. This is the main benefit of using OpenStack cloud as a service (IaaS): it frees you to focus on your own business applications, rather than on maintaining the cloud.
+2. **As a service**: OpenStack as a service means that you purchase your (private) cloud services on contract from experienced cloud operators, such as Bluemix on OpenStack or Rackspace. Cloud operators provide support to their clients, so that you are free to work “with” OpenStack, not “on” OpenStack. This is the main benefit of using OpenStack cloud as a service (IaaS): it frees you to focus on your own business applications, rather than on maintaining the cloud.
```

Comment:
Please delete "Bluemix on OpenStack" and just include the phrase "experienced cloud operators, such as IBM or Rackspace" in this sentence.